apps/challenges/views.py

```diff
@@ -1807,6 +1808,13 @@ def download_all_submissions(
                 # Issue: "#" isn't parsed by writer.writerow(), hence it is replaced by "-"
                 # TODO: Find a better way to solve the above issue.
                 for submission in submissions.data:
+                    if submission["submission_metadata"] is None:
+                        submission["submission_metadata"] = []
+                    submission["submission_metadata"] = ",".join(
+                        f'{key}: {value}'
+                        for metadata in submission["submission_metadata"]
+                        for key, value in metadata.items()
```

Comment:
@afif-fahim Sorry for the delay. We only need to fetch field name and field value. Current changes are giving all metadata sent by frontend in this field. This would be difficult for hosts to interpret. Here's what the current output looks like 

```
"name: TextAttribute,type: text,value: None,required: False,$$hashKey: object:201,description: Sample,name: SingleOptionAttribute,type: radio,value: B,options: ['A', 'B', 'C'],$$hashKey: object:202,description: Sample,name: MultipleChoiceAttribute,type: checkbox,values: ['gamma'],options: ['alpha', 'beta', 'gamma'],$$hashKey: object:203,description: Sample,name: TrueFalseField,type: boolean,value: False,required: True,$$hashKey: object:204,description: Sample"
```

It should be:
```
TextAttribute: None, SingleOptionAttribute: B

```

where `TextAttribute` is the name field in each metadata entry and value for each field name should be value from the stored object. Let me know if you have any doubts
```

Reply:
@Ram81, Sorry for the misunderstanding! I have pushed the new changes.